.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env"]
+}
```

Comment:
Can this be moved to package.json? Example: https://github.com/sindresorhus/refined-github/blob/3737e091b697bf87f0ec4964eb726ab8eaf5ad10/package.json#L86-L103